### PR TITLE
Chore/update qa dcp

### PR DESCRIPTION
**Pull request by atharvar28**
- Updating qa-dcp services to 2021.02 release version
- Testing index-scoped tiered-access (data-portal and guppy changes)

### CODE REVIEW

**Review by themarcelor — APPROVED** [review]
+1

File: qa-dcp.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -7,27 +7,27 @@
...
+    "portal": "quay.io/cdis/data-portal:feat_index-scoped-tiered-access",
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
-    "spark": "quay.io/cdis/gen3-spark:integration202102",
-    "tube": "quay.io/cdis/tube:integration202102",
-    "guppy": "quay.io/cdis/guppy:integration202102",
-    "sower": "quay.io/cdis/sower:integration202102",
-    "hatchery": "quay.io/cdis/hatchery:integration202102",
+    "spark": "quay.io/cdis/gen3-spark:2021.02",
+    "tube": "quay.io/cdis/tube:2021.02",
+    "guppy": "quay.io/cdis/guppy:2021.02",
```
Comment (ZakirG) [inline qa-dcp.planx-pla.net/manifest.json:22]: one more change: guppy should be on the branch `feat_per-index-tiered-access`

File: qa-dcp.planx-pla.net/portal/gitops.json [hunk 2]
```diff
@@ -325,6 +325,7 @@
     },
     "guppyConfig": {
       "dataType": "subject",
+      "tierAccessLevel": "libre",
```
Comment (ZakirG) [inline qa-dcp.planx-pla.net/portal/gitops.json:328]: this file doesn't need changes -- testing should be against site-wide tiered access

**Review by ZakirG — APPROVED** [review]

**Review by themarcelor — APPROVED** [review]
+1

**Comment by PlanXCyborg** [comment]
# qa-dcp.planx-pla.net/manifest.json
## :warning: Services on branch
- portal
- guppy

**Pull request merged**